cmd/diff_test.go

```diff
@@ -13,7 +13,8 @@ var diffArgNumTests = []testpair{
 
 func TestDiffArgNum(t *testing.T) {
 	for _, test := range diffArgNumTests {
-		valid, err := checkDiffArgNum(test.input)
+		err := checkDiffArgNum(test.input)
```

Comment:
This is a bit weird of a construct. Do we need err and valid? Could you do something like:

valid := (checkDiffArgNum == nil) ?

Reply:
good point, changed